hbw/app/assets/stylesheets/hbw/brand_variables.sass

```diff
@@ -14,3 +14,34 @@
   --base-btn-success-color: #{$btn-success-color}
   --base-btn-info-color: #{$btn-info-color}
   --base-btn-warning-color: #{$btn-warning-color}
+
+  //borders
+  --base-hbw-border-light-gray: 1px solid var(--hbw-light-gray, var(--base-hbw-light-gray))
+  //tasks
+  --base-hbw-inactive-task-hover-color: $hbw-inactive-task-hover-color
+  //colors
+  --base-hbw-light-gray: #ECF0F1
+  --base-hbw-red: red
+  --base-hbw-white: white
+  --base-hbw-scrollbar-color: grey white
+  --base-hbw-gray: #888
+  --base-hbw-dark-gray: #555
+  --base-hbw-blue-bayoux: #445C72
```

Comment:
`--base-hbw-scrollbar-color` is good naming, but `--base-hbw-blue-bayoux` is meaningless. We should use domain-related or element-related names for these variables.